playwright-tests/tests/intents/intents-payment-request-detail.spec.js

```diff
@@ -1,11 +1,34 @@
 import { test } from "../../util/test.js";
 import { expect } from "@playwright/test";
-import { redirectWeb4, getLocalWidgetContent } from "../../util/web4.js";
+import { redirectWeb4 } from "../../util/web4.js";
+
+test.use({
+  viewport: {
+    width: 1280,
+    height: 800,
+  },
+});
```

Comment:
is there any specific reason for this?

Reply:
yes this is for making sure that all the content is visible in the video without having to scroll.